go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/lightninglabs/gozmq
+
+go 1.12
```

Comment:
why is this defined here?

Reply:
See https://github.com/golang/go/issues/23969 and https://go-review.googlesource.com/c/go/+/125940/.